xaitk_saliency/interfaces/gen_descriptor_sim_sal.py

```diff
@@ -55,10 +56,10 @@ def generate(
 
         :param ref_descr:
             Reference image float feature-vector, shape `[nFeats]`
-        :param query_descr:
-            Query image float feature-vector, shape `[nFeats]`.
+        :param query_descrs:
+            Query images float feature-vectors, shape `[nQueryImgs x nFeats]`.
```

Comment:
The double-plural sounds odd to me, but I'm happy for someone for fact-check my "grammar feel".
```suggestion
            Query image float feature-vectors, shape `[nQueryImgs x nFeats]`.
```